src/goto-programs/Makefile

```diff
@@ -19,6 +19,7 @@ SRC = goto_convert.cpp goto_convert_function_call.cpp \
       class_hierarchy.cpp show_goto_functions.cpp get_goto_model.cpp \
       slice_global_inits.cpp goto_inline_class.cpp class_identifier.cpp \
       show_goto_functions_json.cpp \
+      remove_const_function_pointers.cpp \
```

Comment:
Cf the discussion on coding guidelines: your insertion point is a random one (which is fine at present), but for the principle of least surprise there should be a canonical insertion point.